rllib/utils/replay_buffers/multi_agent_episode_buffer.py

```diff
@@ -285,11 +314,39 @@ def add(
             else:
                 # New episode.
                 self.episodes.append(eps)
+                # Update the counters
+                num_episodes_added += 1
+                for aid, a_eps in eps.agent_episodes.items():
+                    mid = eps._agent_to_module_mapping[aid]
+                    agent_to_num_episodes_added[aid] += 1
+                    module_to_num_episodes_added[mid] += 1
                 eps_idx = len(self.episodes) - 1 + self._num_episodes_evicted
                 self.episode_id_to_index[eps.id_] = eps_idx
                 self._indices.extend([(eps_idx, i) for i in range(len(eps))])
                 # Add new module indices.
                 self._add_new_module_indices(eps, eps_idx, False)
+            # Update the step counters.
+            num_env_steps_added += eps.env_steps()
+            for aid, e_eps in eps.agent_episodes.items():
+                mid = eps._agent_to_module_mapping[aid]
+                agent_to_num_steps_added[aid] += e_eps.agent_steps()
+                module_to_num_steps_added[mid] += e_eps.agent_steps()
+
+        # Update the adding metrics.
+        self._update_add_metrics(
```

Comment:
Just for safety: Can we make all these utility methods with forced keywords?

```
def _update_add_metrics(self, *, ...):
```

Also rename to `_log_metrics`.

Reply:
Sure. Great idea.